layouts/partials/js.html

```diff
@@ -1,8 +1,16 @@
-<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-<script src="{{ .Site.BaseURL }}js/scrollreveal.min.js"></script>
-<script type='application/javascript' src='{{ .Site.BaseURL }}js/smooth-scrollbar.js'></script>
-<script src="{{ .Site.BaseURL }}js/baffle.min.js"></script>
-<script src="{{ .Site.BaseURL }}js/main.js"></script>
+<script>
+(function () {
+  if (window.fetch) return;
+  document.write('<script src="https://cdn.jsdelivr.net/jquery/3.2.1/jquery.slim.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/scrollreveal.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/smooth-scrollbar.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/baffle.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/baffle.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/main.js"><\/script>');
+})()
+</script>
```

Comment:
This is a fallback for older browsers which do not support `window.fetch`. Note I updated URLs here and elsewhere to load from HTTPS always to help prevent man in the middle xss attacks on jQuery. Also switched CDNs as jsDelivr is open source and very hackable.

For more info on sub-resource integrity checking please see: http://devdocs.io/html/attributes#integrity-attribute and check out the H5BP.